### PR TITLE
RD-1122 Make summary tests py3 compat.

### PR DESCRIPTION
**Pull request by YoniItzhak**

### CODE REVIEW

**Review by tehasdf — APPROVED** [review]

File: cosmo_tester/test_suites/summary/summary_test.py [hunk 1]
```diff
@@ -551,20 +511,17 @@ def test_subfield_summary_nodes(prepared_manager):
...
+    _assert_summary_equal(results, expected)
 
 
 def test_subfield_summary_node_instances(prepared_manager):
-    results = _sort_summary(prepared_manager.client.summary.node_instances.get(
+    results = prepared_manager.client.summary.node_instances.get(
         _target_field='node_id',
-        _sub_field='state',
-    ).items, sort_key=['node_id', 'state'])
+        _sub_field='state',).items
```
Comment (geokala) [inline cosmo_tester/test_suites/summary/summary_test.py:522]: The hanging comma here looks a bit ugly

File: cosmo_tester/test_suites/summary/summary_test.py [hunk 2]
```diff
@@ -404,26 +365,28 @@ def test_subfield_summary_deployments(prepared_manager):
         {
             u'by blueprint_id': [
                 {u'blueprint_id': u'multivm', u'deployments': 3},
-                {u'blueprint_id': u'small', u'deployments': 1},
-                {u'blueprint_id': u'relations', u'deployments': 2}
+                {u'blueprint_id': u'relations', u'deployments': 2},
```
Comment (geokala) [inline cosmo_tester/test_suites/summary/summary_test.py:368]: Any reason for these being reordered here?

File: cosmo_tester/test_suites/summary/summary_test.py (hunk 1 above)
Comment (YoniItzhak) [reply cosmo_tester/test_suites/summary/summary_test.py:522]: Sure, deleted

File: cosmo_tester/test_suites/summary/summary_test.py (hunk 2 above)
Comment (YoniItzhak) [reply cosmo_tester/test_suites/summary/summary_test.py:368]: This way the list is ordered by `blueprint_id`, as the `results` list.

Comment (geokala) [reply cosmo_tester/test_suites/summary/summary_test.py:368]: Fair enough.

**Review by geokala — APPROVED** [review]

**Pull request merged**